src/content/docs/en/guides/cms/strapi.mdx

```diff
@@ -293,23 +289,23 @@ const article = Astro.props;
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <title>{article.attributes.title}</title>
+    <title>{article.title}</title>
   </head>
 
   <body>
     <main>
-      <img src={import.meta.env.STRAPI_URL + article.attributes.image.data.attributes.url} />
+      <img src={import.meta.env.STRAPI_URL + article.image.data.url} />
```

Comment:
admittedly, I don't know for *sure* that I can just remove two `.attributes` like this? Is it safe to assume if one is gone, then they've also updated the shame of image data?